SUMMARY.md

```diff
@@ -6,6 +6,20 @@
     * [Data Sources](data-sources.md)
     * [Software](software.md)
 * [User Manual](user-manual.md)
+    * [Data Products](data-products.md)
+        * [Infrared head imaging](flir.md)
+        * [Template](template.md)
+        * Photochemical reflectance index
```

Comment:
the PRI and NDVI sensors could be merged into a single file called "Multispectral"